src/tracks/playabletrack/wavetrack/ui/WaveClipAdjustBorderHandle.cpp

```diff
@@ -177,19 +164,12 @@ class AdjustClipBorder final : public WaveClipAdjustBorderHandle::AdjustPolicy
       {
          for(const auto track : as_const(*trackList))
          {
-            const auto isSameTrack = (track == currentTrack) ||
-               (track->GetLinkType() == Track::LinkType::Aligned && *trackList->Find(currentTrack) == track) ||
-               (currentTrack->GetLinkType() == Track::LinkType::Aligned && *trackList->Find(track) == currentTrack);
             for(const auto& interval : track->Intervals())
             {
-               if(isSameTrack)
+               if(track == currentTrack)
                {
-                  auto waveTrackIntervalData =
-                     std::dynamic_pointer_cast<const WaveTrack::Interval>(
-                        interval);
-                  if(waveTrackIntervalData->GetClip(0).get() == adjustedClip)
-                  //exclude boundaries of the adjusted clip
-                     continue;
+                  //skip track that interval belongs to
+                  continue;
```

Comment:
Should this test be lifted out of this inner loop?

This behavior change ignores all intervals of the current track, unlike the previous version which ignored only one interval, but this makes sense:  other clips would limit the movement of the selected clip and their boundaries would not be useful snap positions.

Reply:
> Should this test be lifted out of this inner loop?

Yes, thanks!